File: api_tests/request.py
import http.client

conn = http.client.HTTPConnection("127.0.0.1:5000")

payload = "------WebKitFormBoundary7MA4YWxkTrZu0gW\r\nContent-Disposition: form-data; name=\"username\"\r\n\r\ntoto\r\n------WebKitFormBoundary7MA4YWxkTrZu0gW\r\nContent-Disposition: form-data; name=\"email\"\r\n\r\ntoto@gmail.com\r\n------WebKitFormBoundary7MA4YWxkTrZu0gW\r\nContent-Disposition: form-data; name=\"password\"\r\n\r\ntoto\r\n------WebKitFormBoundary7MA4YWxkTrZu0gW\r\nContent-Disposition: form-data; name=\"confirm\"\r\n\r\ntoto\r\n------WebKitFormBoundary7MA4YWxkTrZu0gW--"

headers = {
    'content-type': "multipart/form-data; boundary=----WebKitFormBoundary7MA4YWxkTrZu0gW",
    'cache-control': "no-cache",
    'postman-token': "58bbf749-ab6d-60b5-f2cd-7aa954d678f1"
    }

conn.request("POST", "/register", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: api_tests/request.php
<?php

$request = new HttpRequest();
$request->setUrl('http://127.0.0.1:5000/register');
$request->setMethod(HTTP_METH_POST);

$request->setHeaders(array(
  'postman-token' => 'db5ec391-2f12-bd7f-ae0e-44d5483bec3b',
  'cache-control' => 'no-cache',
  'content-type' => 'multipart/form-data; boundary=----WebKitFormBoundary7MA4YWxkTrZu0gW'
));

$request->setBody('------WebKitFormBoundary7MA4YWxkTrZu0gW
Content-Disposition: form-data; name="username"

toto
------WebKitFormBoundary7MA4YWxkTrZu0gW
Content-Disposition: form-data; name="email"

toto@gmail.com
------WebKitFormBoundary7MA4YWxkTrZu0gW
Content-Disposition: form-data; name="password"

toto
------WebKitFormBoundary7MA4YWxkTrZu0gW
Content-Disposition: form-data; name="confirm"

toto
------WebKitFormBoundary7MA4YWxkTrZu0gW--');

try {
  $response = $request->send();

  echo $response->getBody();
} catch (HttpException $ex) {
  echo $ex;
}